src/main/java/util/DataConverterImpl.java

```diff
@@ -0,0 +1,26 @@
+package util;
+
+import java.util.ArrayList;
+import java.util.List;
+import model.FruitTransaction;
+
+public class DataConverterImpl implements DataConverter {
+    @Override
+    public List<FruitTransaction> convertToTransaction(List<String> lines) {
+        List<FruitTransaction> transactions = new ArrayList<>();
+        for (String line : lines) {
+            String[] parts = line.split(",");
+            FruitTransaction.Operation operation = FruitTransaction.Operation.fromCode(parts[0]);
+            String fruit = parts[1];
+            int quantity = Integer.parseInt(parts[2]);
```

Comment:
Direct parsing of the quantity from a string without any validation or error handling can lead to `NumberFormatException` if the string is not a valid integer. Consider adding error handling to provide a more informative error message in case of invalid input.